Tools-Override/tests.targets

```diff
@@ -0,0 +1,162 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
When we start adding some of these targets can I suggest we have one commit that is unchanged and then a follow-up commit which has the changes so we can see the delta of what we changed from buildtools?